app/views/content/non-uk-teachers/visas-for-non-UK-trainees.md

```diff
@@ -0,0 +1,257 @@
+---
+title: "Apply for your visa to train to teach in England"
+description: |-
+  Learn more about types of visa non-UK citizens will need to study or work in England, and how to apply.
+related_content:
+    Career progression stories: "/blog/tag/career-progression"
+    Salaries and benefits of teaching in England : "/salaries-and-benefits"
+    A day in the life of a teacher in England : "/a-day-in-the-life-of-a-teacher"
+promo_content:
+date: "2021-05-27"
+image: "static/content/hero-images/0002.jpg"
+backlink: "../../"
+keywords:
```

Comment:
This list seems very long with some quite general terms. I wonder if it would be better to have fewer more specific terms here. @EllieNodder what's your thoughts on this list?

Reply:
I'd agree, I think! Not sure how helpful the terms like 'PhD' and 'Abroad' will be?

Reply:
I didn't actually create this list – not sure what the thinking behind Korea and Taiwan is for example – please feel free to edit :)